README.md

```diff
@@ -8,142 +8,30 @@
-                "parameters": {
-                    "labels": [
-                        "audi",
-                        "beetle"
-                    ],
-                    "probs": [
-                        0.2,
-                        0.8
-                    ]
-                }
-            }
-        },
-        {
-            "name": "optional",
-            "type": "discrete",
-            "dtype": "Int64",
-            "prop_missing": 0.2,
-            "distribution": {
-                "implements": "core.discrete_uniform",
-                "provenance": "builtin",
-                "class_name": "DiscreteUniformDistribution",
-                "parameters": {
-                    "low": -30,
-                    "high": 301
-                }
-            }
-        }
-    ]
-}
-```
-
-A more advanced example GMF, based on the [Titanic](https://raw.githubusercontent.com/pandas-dev/pandas/main/doc/data/titanic.csv) dataset, can be found [here](examples/titanic_example.json)
-</details>
+The package has three main functionalities:
 
-### Generating synthetic data from a GMF file
-MetaSynth can then be used to **generate synthetic data** from any GMF standard .JSON file.
+1. **Estimation**: MetaSynth can create a MetaFrame, from a dataset. A MetaFrame is essentially a fitted model that characterizes the structure of the original dataset without storing actual values. It captures individual distributions and features, enabling generation of synthetic data based on these MetaFrames and can be seen as (statistical) metadata.
+2. **Serialization**: MetaSynth can export a MetaFrame into an easy to read JSON file, allowing users to audit, understand, and modify their data generation model.
+3. **Generation**: MetaSynth can generate synthetic data based on a MetaFrame. The synthetic data produced solely depends on the MetaFrame, thereby maintaining a critical separation between the original sensitive data and the synthetic data generated. The generated synthetic data, emulates the original data's format and plausibility at the individual record level and attempts to reproduce marginal (univariate) distributions where possible. Generated values are based on the observed distributions while adding a degree of variance and smoothing. The generated data does **not** aim to preserve the relationships between variables. The frequency of missing values and their codes are maintained in the synthetically-augmented dataset. 
 
-![Synthetic_data_generation](docs/source/images/flow_synthetic_data_generation.png)
+Researchers and data owners can use MetaSynth to generate and share synthetic versions of their sensitive datasets, mitigating privacy concerns. Additionally, MetaSynth facilitates transparency and reproducibility, by allowing the underlying MetaFrames to be exported and shared. Other researchers can use these to regenerate consistent synthetic datasets, validating published work without requiring sensitive data.
```

Comment:
I think I would rather have it at before the list, since it seems the most important part of the README.